### ISSUE TITLE
Frictionless accessibility fixes

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1331]
- [x] The "View Issues" and "Remove" controls are coded as links, but should be buttons.
- [x] The blinking "Checking..." instead of the progress bar is not ideal. I don't think it's a big deal under these circumstances, but blinking text is super-distracting for ADHD folks. If it's not feasible to actually track the progress of the check, I'm sure we can come up with a less distracting animation to indicate that work is in progress.
- [x] The green "Passed" state is from an older wireframe. Again, not a big deal, but I replaced it with "No issues" in black because the green seemed a little too attention-grabbing. My general thought was that if your dataset passes, you don't really need to notice or think about it, so I wanted the passed status to fade into the rest of the table. I changed the text from "passed" to "no issues" to avoid a pass/fail dynamic.
- [x] The current wireframe has the number of issues in the button text (e.g., "View 3 issues"). I think this would be nice to know, but isn't critical.
- [x] The warning icons are missing from the table.
